### DISCUSSION TITLE
Error de compilación

### DISCUSSION THREAD

**Discussion by mauromunilla** [dds-jv/foro#374] [Consultas generales] [answered]
![image](https://user-images.githubusercontent.com/63859726/170881463-f9755cf7-2de9-4458-ba45-a4d50baba2ba.png)

En el IDE me funciona perfecto, pero me salta error "cannot find symbol" en el Actions

**Comment by gastonprieto** [comment, marked as answer]
buenas @mauromunilla 

el tag lo hiciste sobre el commit inicial, el tag siempre se hace una vez q tenes todo commiteado y pusheado a github y chequeas q en github esta todo.

**Reply by mauromunilla** [reply]
Ah, ok. Entonces una vez hecho ya no lo puedo solucionar, no?

**Reply by gastonprieto** [reply]
si lo podes solucionar, buscate en el foro que varios pusieron como lo resolvieron
